src/lp_matrix_data.jl

```diff
@@ -53,6 +56,10 @@ The struct returned by [`lp_matrix_data`](@ref) has the fields:
  * `c::Vector{T}`: the dense vector of linear objective coefficiennts
  * `c_offset::T`: the constant term in the objective function.
  * `sense::MOI.OptimizationSense`: the objective sense of the model.
+ * `integers::Vector{Int}`: the list of 1-indexed columns that are integer
```

Comment:
> 1-indexed columns
This confused me when I first read it. Is it needed? That is, aren't all lists in Julia indexed starting from 1?
I think this comment means 'a vector of column indices for A [alt.: the constraint matrix] that match up to integer variables' (sim. for binaries)